tcms/testruns/static/testruns/js/get.js

```diff
@@ -162,6 +165,19 @@ $(document).ready(() => {
     testExecutionSelectors.each((_index, te) => { te.checked = isChecked })
   })
 
+  $('.assigned-to-me-checkbox').click(({ target }) => {
+    const isChecked = target.checked
+    const currentUser = $('#test_run_pk').data('current-user')
+
+    const filterValue = isChecked ? currentUser : ""
+    filterTestExecutionsByProperty(
+      testRunId,
+      Object.values(displayedExecutions),
+      "assignee__username",
+      filterValue
+    )
+  })
```

Comment:
There is a small bug here. If I filter the page then change the assignee of one of the visible test cases and click on/off on the button then the TC which had its assignee changed is still visible even though that's a different user.  The rest seems to work.

Reply:
I fixed the other things. Will take a look at this tomorrow.